api/namex/models/request.py

```diff
@@ -326,6 +326,23 @@ def find_name_by_choice(cls, nr_id, choice):
         name_by_choice = next((name for name in names if name.choice == choice), None)
         return name_by_choice
 
+    @classmethod
+    def find_existing_name_by_user(cls, user_name_search_string, request_owner):
```

Comment:
because it is a email presenting the request_owner, could you change the parameter's name to email or such for less confusing.

Reply:
updated